ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Combinator/State/Types.hs

```diff
@@ -93,10 +95,18 @@ newtype TranslateForecast f g x y = TranslateForecast {
       translateForecastWith ::
            Bound    -- 'Bound' of the transition (start of the new era)
         -> SlotNo   -- 'SlotNo' we're constructing a forecast for
-        -> f x
+        -> f x Canonical
         -> Except OutsideForecastRange (Ticked (g y))
     }
 
+-- | Translate a LedgerState across an era transition.
+data TranslateLedgerState x y = TranslateLedgerState {
```

Comment:
```suggestion
newtype TranslateLedgerState x y = TranslateLedgerState {
```

Reply:
Although this is absolutely true, it is going to become a `data` very soon because it gets another field, so I think we can just leave it as is?